src/include/pbs_ifl.h

```diff
@@ -360,6 +361,7 @@ extern "C" {
 #define ATTR_sched_preempt_order  "preempt_order"
 #define ATTR_sched_preempt_sort  "preempt_sort"
 #define ATTR_sched_server_dyn_res_alarm "server_dyn_res_alarm"
+#define ATTR_job_run_wait "job_run_wait"
```

Comment:
Since the functions are called pbs_runjob() or or similar, should this be runjob_wait?

Reply:
this was a design discussion point, Scott had mentioned that he preferred job_run_wait instead of runjob_wait because we already have an existing association of the word "runjob" with hooks.

Reply:
Thanks for reminding me.  I thought there must be a reason.  You may want to rename your tests since they are test_runjob_wait_X